apps/src/lib/ui/PopUpMenu.jsx

```diff
@@ -46,6 +47,37 @@ class PopUpMenu extends Component {
     }).isRequired,
     children: PropTypes.any,
     className: PropTypes.string,
+    isOpen: PropTypes.bool,
+    beforeClose: PropTypes.func,
+  };
+
+  render() {
+    return (
+      <Portal
+        closeOnEsc
+        closeOnOutsideClick
+        isOpened={this.props.isOpen}
+        beforeClose={this.props.beforeClose}
+      >
+        <RadiumMenuBubble
+          targetPoint={this.props.targetPoint}
+          className={this.props.className}
+          children={this.props.children}
+        />
+      </Portal>
+    );
+  }
+}
+export default PopUpMenu;
+
+class MenuBubble extends Component {
```

Comment:
Although personally, I think decorator syntax is really the best:

```jsx
@Radium
export default class MenuBubble extends Component {
}
```

Sadly they removed the decorator syntax from the draft spec :( :(